packages/patternfly-4/react-docs/src/components/componentDocs/componentDocs.js

```diff
@@ -57,6 +58,7 @@ class ComponentDocs extends React.PureComponent {
     } = this.props;
     const makeDescription = html => ({ __html: html });
     const getDocGenInfo = name => data.allComponentMetadata.edges.find(edge => edge.node.displayName === name);
+    const getDocGenInfoTs = name => data.allTsDocsJson.edges[0].node.data.find(edge => edge.name === `${name}Props`);
```

Comment:
Right now making the assumption to find the corresponding prop for a component by appending `Props` to the component name, e.g. Avatar => AvatarProps